src/base/bittorrent/session.cpp

```diff
@@ -3762,7 +3762,7 @@ void Session::readAlerts()
         handleAlert(a);
 }
 
-void Session::handleAlert(libt::alert *a)
+void Session::handleAlert(const libtorrent::alert *a)
```

Comment:
why drop the namespace shorthand?

Reply:
That's weird because i didn't... i'll fix.